src/web/src/mixins/NotificationsMixin.js

```diff
@@ -48,5 +48,17 @@ export default {
         noAutoHide: false,
       });
     },
+    notifyCreditConflict(){
+      //Format All Message Data For Conflict Message:
+      const vNodesMsg = this.$createElement("p", { class: ["mb-0"] }, [
+        'WARNING: Exceeding Credits Limit'
+        ]);
+      //Format All Header-Data For Conflict Message:
+      this.$bvToast.toast(vNodesMsg, {
+        title: `CREDIT CAP`,
+        variant: "danger",
+        noAutoHide: false,
+      });
```

Comment:
Perhaps we link to admin panel configuration with RPI link? i.e. we configure in admin panel to link to http://catalog.rpi.edu/content.php?catoid=4&navoid=83.

So that In admin panel, we can set 1. credit warning limit 2. warning content

wyt?